sdk/python/feast/repo_config.py

```diff
@@ -242,7 +242,17 @@ def __init__(self, **data: Any):
 
         self._online_store = None
         if provider == "expedia":
-            self.online_config = data.get("online_store", "redis")
+            self.online_config = data.get(
+                "online_store",
+                {
+                    "type": "redis",
+                    "connection_string": "${REDIS_CONNECTION_STRING}",
+                    "redis_type": "redis_cluster",
+                    "key_ttl_seconds": 604800,
+                },
+            )
```

Comment:
```suggestion
            self.online_config = data.get("online_store", "redis")
```

Reply:
Is online_config is string, it uses the defaults set. Check online_store property method below.